PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+<!-- 
+Thanks for opening a pull request! 🎉
+
+If your changes include some kind of UI element, please attach a screenshot of 
+how it looks.
+
+Before submitting a pull request, please have a look through the CONTRIBUTING.md
+document. TL;DR:
+
+- Write proper commit messages tagged with the correct package and scope
```

Comment:
Mention conventional commits specifically here, perhaps?

Reply:
Sure. Do you have a suggestion as to how it should be worded?

Reply:
> Write proper commit messages tagged with the correct package and scope. We follow the [Conventional Commits](https://conventionalcommits.org/) convention.

... or some such. :)